LiveChat/components/Chat.js

```diff
@@ -48,15 +48,22 @@ export default class Chat extends React.Component {
 		if (isChatOn) {
 			return (
 				<View
-					animation="fadeInUp"
+					animation='fadeInUp'
 					style={styles.container}
 					ref={(ref) => {
 						this.chat = ref
 					}}
 				>
-					<NavigationBar chatTitle={chatTitle} closeChat={closeChat} />
-					{isReconnecting && <Text style={styles.connectionStatus}>Reconnecting...</Text>}
-					{headerText && <Text style={styles.status}>{headerText}</Text>}
+					{this.props.NavBarComponent ? (
+						<this.props.NavBarComponent chatTitle={chatTitle} closeChat={closeChat} isReconnecting={isReconnecting}
+																				headerText={headerText} />
```

Comment:
Formatting ended up a bit strange here, but not a big deal

Reply:
It did, I used the built in formatted, I'm not sure why it decided it needed to move that last one so far out though.  I made a fix though, da00882